internal/locate/region_request.go

```diff
@@ -2211,7 +2222,11 @@ func (s *RegionRequestSender) onSendFail(bo *retry.Backoffer, ctx *RPCContext, r
 			}
 		}
 	}
-	metrics.TiKVRPCErrorCounter.WithLabelValues(errors.Cause(err).Error(), storeLabel).Inc()
+	if errStr := getErrMsg(err); len(errStr) > 0 {
+		metrics.TiKVRPCErrorCounter.WithLabelValues(getErrMsg(err), storeLabel).Inc()
```

Comment:
will the error message be too long?

Reply:
It's possible, but I haven't seen any error message very long yet.

Reply:
my issue error message is:


![image](https://github.com/tikv/client-go/assets/6428910/8e6774ea-b457-44d6-b629-d2a289603f0a)

Reply:
Great, I think this error length is acceptable.